windows/RNTrackPlayer/Players/Playback.cs

```diff
@@ -61,6 +61,18 @@ public abstract class Playback {
             manager.OnTrackUpdate(previous, position, track, true);
         }
 
+        protected void OnEnd(MediaPlayer sender, object args) {
+            if(HasNext()) {
+                UpdateCurrentTrack(currentTrack + 1, null);
+                Play();
+            } else {
+                Track previous = GetCurrentTrack();
+                double position = GetPosition();
+
+                manager.OnEnd(previous, position);
+            }
+        }
+
```

Comment:
Why did you moved this function here? It accepts a MediaPlayer as an argument because it is an event listener for the local playback.

Reply:
OK. I'll give it another go.